.github/workflows/candidate_to_master.yml

```diff
@@ -0,0 +1,22 @@
+name: Create pull request from candidate to master
+on:
+  release:
+    types: [published]
+jobs:
+  candidateToMaster:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: master
+      - name: Reset candidate
+        run: |
+          git fetch origin candidate:candidate
+          git reset --hard candidate
```

Comment:
this reset is very dangerous, it will remove all commits done on master after the candidate branch was created, right?

Reply:
Yeah, thank you for your concern. It could be if the files are wrongly modified / deleted from the git history either in master or candidate branch.  So, I've checked the output in the git history in my side, it looks fine.